addons/l10n_it_edi_withholding/tests/test_withholding.py

```diff
@@ -380,3 +415,24 @@ def test_enasarco_tax_import(self):
             self.assertEqual(self.enasarco_purchase_tax, enasarco_imported_tax)
             self.assertEqual(-8.5, enasarco_imported_tax.amount)
             self.assertEqual(self.withholding_purchase_tax_23, line.tax_ids.filtered(lambda x: x.l10n_it_withholding_reason == 'ZO'))
+
+    def test_inps_tax_export(self):
+        """
+            Invoice
+            -----------------------------------------------------------------
+            Ordinary accounting service for the year                   350.00
+            Balance deposit for the past year                          300.00
+            Ordinary accounting service for the trimester               50.00
+            Electronic invoices management                              50.00
+            -----------------------------------------------------------------
+            Total untaxed:                                             750.00
+            VAT:             0% of Untaxed Amount                        0.00
+            INPS:            4% of Untaxed Amount                       30.00
+            Document total:  Taxed Amount                              780.00
+            Payment amount:  Document total                            780.00
+        """
+        content = self.edi_format._l10n_it_edi_export_invoice_as_xml(self.inps_tax_invoice)
+        cleanups = {'.//FatturaElettronicaBody/DatiPagamento': 'DatiPagamento'}
+        expected_etree = self._cleanup_etree(self.edi_inps_tax_xml, cleanups)
+        invoice_etree = self._cleanup_etree(content, cleanups)
```

Comment:
I find it weird in the test to both clean the expected_xml and the one generated in the invoice.  Why not remove it from the sample XML file then?  (and somehow I don't see how <xpath path="..." position="replace">DatiPagamento</xpath> works

Reply:
👍🏻  I'll cleanup the test XML file 
DatiPagamento gets cleaned up in _cleanup_etree like the rest, it's added to the removal of Allegati and the Progressivo by that `cleanup` parameter.